play/src/front/Chat/Connection/Proximity/ProximityChatRoom.ts

```diff
@@ -461,6 +467,21 @@ export class ProximityChatRoom implements ChatRoom {
         this.simplePeer.setSpaceFilter(undefined);
     }
 
+    private restoreChatState() {
+        if (get(selectedRoomStore) == this) {
+            selectedRoomStore.set(this.currentMatrixRoom);
+        }
+
+        chatVisibilityStore.set(this.currentChatVisibility);
```

Comment:
You should restore the chat state only if the user did not navigate away from the proximity chat.

If the user left the proximity chat while in the bubble, we should probably do nothing.